app/mailers/student_mailer.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+#
+# Mailer class for user e-mails
+class StudentMailer < ApplicationMailer
+  default from: 'no-reply@vesta.app' # SHOULD WE HAVE AN ADMIN ADDRESS?
+
+  def draw_invitation(user)
+    @user = user
+    @intent_deadline = user.draw.intent_deadline
+    # TODO: figure this out - singleton class, e.g. app config, or do we add a
+    # class for now that we later use for multitenancy?
```

Comment:
Yea exactly. I think that some kind of high-level model for the root-level user (similar to `Department` in Shifts) will work fine.